app/helpers/dashboards_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module DashboardsHelper
+  def latest_upload_class(upload)
+    return '' if upload.ok?
+    return 'danger' if CSV_TYPES_REQUIRED_TABLE_NAMES.include?(upload.csv_type)
+    return 'warning' unless CSV_TYPES_REQUIRED_TABLE_NAMES.include?(upload.csv_type)
```

Comment:
The `unless` statement is not necessary.  If we make it to this line `CSV_TYPES_REQUIRED_TABLE_NAMES` does not include `upload.csv_type` since we checked for it's presence in the previous line.
```suggestion
    return 'warning'
```

Reply:
changed